lib/Driver/Driver.cpp

```diff
@@ -1755,8 +1757,17 @@ Driver::computeCompilerMode(const DerivedArgList &Args,
     return Inputs.empty() ? OutputInfo::Mode::REPL
                           : OutputInfo::Mode::Immediate;
 
-  const Arg *ArgRequiringSingleCompile = Args.getLastArg(
-      options::OPT_whole_module_optimization, options::OPT_index_file);
+  bool HasWMO = Args.hasFlag(options::OPT_whole_module_optimization,
```

Comment:
This variable name feels very slightly misleading, because we may _have_ received the WMO flag but we may not _apply_ WMO to the module (or _use_ WMO mode), because of a negated flag. What do you think about naming it `ApplyWMO` or `UseWMO` instead?

Reply:
Updated to `UseWMO`